packages/components/package.json

```diff
@@ -118,7 +118,8 @@
   "devDependenciesComments": {
     "postcss": "Installed in root",
     "postcss-cli": "Installed in root",
-    "postcss-preset-env": "Installed in root"
+    "postcss-preset-env": "Installed in root",
+    "react-day-picker": "Pinned to v8, v9 causes typescript errors"
```

Comment:
1. Already a comment in L95 :)
2. When did it get unlocked to begin with 😂 Nonetheless, we should actually be trying to upgrade to v9 and fixing our code because they're meant to have fixed the a11y issues they introduced prior.

Reply:
In another PR